book/sourdough-starter/sourdough-starter-types.tex

```diff
@@ -117,42 +117,42 @@ \section{Liquid starter}
   \includegraphics{figures/fig-liquid-starter-conversion.pdf}
   \caption{The process to convert your regular or stiff starter into a liquid starter. The whole
   process takes around 3 days. The longer you maintain your starter at the
-  suggested hydration level the more adapted your microorganisms become. It is recommended
+  suggested hydration level, the more adapted your microorganisms become. It is recommended
   to keep a backup of your original starter as the liquid environment will select
-  an-aerobic microorganisms. This boosts bacteria that create lactic acid rather
+  anaerobic microorganisms. This boosts bacteria that create lactic acid rather
   than acetic acid. The resulting acidity will be perceived as milder.}
   \label{fig:liquid-starter-conversion}
 \end{figure}
 
 The liquid starter is made at a hydration of around 500 percent. This means
-the starter has way more water than flour. The additional layer of water on
+the starter has much more water than flour. The additional layer of water on
 top of the flour changes the microbiome of your starter.
 
-By introducing this layer of water less oxygen is available throughout the
+By introducing this layer of water, less oxygen is available throughout the
 course of fermentation. This means that your starter will no longer be
 producing acetic acid. The heterofermentative lactic acid bacteria will thrive
 in this environment. This is a neat little trick to change your starter's
-flavour profile from vinegary to lactic. Your starter is going to develop
-dairy creamy notes. Interesting when changing the hydration again your starter
+flavor profile from vinegary to lactic. Your starter is going to develop
+dairy creamy notes. Interestingly, when changing the hydration again, your starter
 is going to maintain the liquid starter flavor profile, but then benefit again
 from enhanced yeast activity. The liquid starter conversion is non reversible.
-So ideally keep a backup of your starter before.
+So ideally keep a backup of your stiff or regular starter.
 
 To commence with the
-conversion simply take around 1 gram of your starter, mix with 5g flour and
-25g water. Stir everything together properly. After a few minutes the flour is
+conversion, simply take around 1 gram of your starter, mix with 5 g flour and
+25 g water. Stir everything together properly. After a few minutes the flour is
 going to start settling in at the bottom of your jar. Repeat this process over
-a few days. Shake the starter gently to see if you can see tiny CO2 bubbles
+a few days. Shake the starter gently to see if you can see tiny CO_{2} bubbles
```

Comment:
The `CO_{2}` expression is causing latex build failure on [Validate Latex Document Check](https://github.com/hendricius/the-sourdough-framework/pull/64/checks?check_run_id=12269500869).

Try substituting `CO_{2}` -> `CO\textsubscript{2}` in this file. This succeeded with my local testing and produced the desired output.

Reply:
Thank you very much for looking into that!  I'll give it a try.